src/extensions/register-page.tsx

```diff
@@ -43,4 +49,15 @@ export class PagesStore {
   }
 }
 
+export class DynamicPage extends React.Component<{ page: PageRegistration }> {
+  render() {
+    const { className, components: { Page }, subPages = [] } = this.props.page;
+    return (
+      <TabLayout className={cssNames("ExtensionPage", className)} tabs={subPages}>
```

Comment:
Well, it could be considered as one of "system" component classNames. Otherwise we have a tons top-level `ComponentName` className which might conflict with extension classNames (depending how they gonna be bundled/loaded with extension).